test/common/stream_info/filter_state_impl_test.cc

```diff
@@ -134,36 +134,91 @@ TEST_F(FilterStateImplTest, WrongTypeGet) {
                             "Data stored under test_name cannot be coerced to specified type");
 }
 
-namespace {
+TEST_F(FilterStateImplTest, IterateThroughListTillEnd) {
+  size_t access_count = 0u;
```

Comment:
`u` suffix isn't needed.

Reply:
copy paste and maintaining consistency throughout this file which has 0us..